projects/core/src/cart/facade/cart.service.spec.ts

```diff
@@ -414,4 +410,58 @@ describe('CartService', () => {
       expect(result).toEqual(cart);
     });
   });
+
+  describe('getAppliedVouchers', () => {
+    it('should return voucher array', () => {
+      const testCart: Cart = <Cart>{
+        appliedVouchers: [{ code: 'voucher1' }, { code: 'voucher2' }],
+      };
+      store.dispatch(new fromCart.LoadCartSuccess(testCart));
+
+      let result: Voucher[];
+      service
+        .getAppliedVouchers()
+        .subscribe(value => (result = value))
+        .unsubscribe();
+      expect(result).toEqual(testCart.appliedVouchers);
+    });
+  });
+
+  describe('add Voucher', () => {
+    it('should be able to addVoucher', () => {
+      spyOn(store, 'dispatch').and.callThrough();
```

Comment:
You can use spyOn(store, 'dispatch').and.stub(); to keep consistent

Reply:
updated